### ISSUE TITLE
build error on windows with gcc on msys64

### ISSUE THREAD

**Issue by ChandanKSingh** [OpenSourceRisk/Engine#116]
There is a liker error while linking libOREData.dll, its not able to link with some regex functions from boost.
![image](https://user-images.githubusercontent.com/7530977/190378621-845defbf-b632-4708-9e94-da9d204b0289.png)

**Comment by ChandanKSingh** [comment]
Hi @pcaspers, yes it points to multiple definition of Constraint. Is it know issue?

**Comment by pcaspers** [comment]
No, I just asked because you were talking about boost regex initially.

**Comment by pcaspers** [comment]
Is there a reason why you build ORE in this way instead of just using MSVC? I don't think we will look into fixing this exotic setup to be honest.
